core/domain/exp_jobs_one_off_test.py

```diff
@@ -250,7 +250,7 @@ def setUp(self):
     def test_first_published_time_of_exploration_that_is_unpublished(self):
         """This tests that, if an exploration is published, unpublished, and
         then published again, the job uses the first publication date as the
-        first published datetime.
+        first published time.
```

Comment:
Well -- it's a bit odd to use a date as a time :)

Maybe: as the first_published_msec time?